apps/strapi-cms/src/components/common/hero.json

```diff
@@ -0,0 +1,24 @@
+{
+  "collectionName": "components_common_heroes",
+  "info": {
+    "displayName": "hero",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "siteTitle": {
+      "type": "string",
+      "required": true
+    },
+    "boldTitle": {
+      "type": "string",
+      "required": true
+    },
```

Comment:
The hero doesn't have a `siteTitle` and a `boldTitle`. I would move both to the Card entity and join them in a single wysiwyg field